.gitignore

```diff
@@ -1,20 +1,9 @@
-## User settings
-xcuserdata/
```

Comment:
Why not include this one?

Reply:
Ah I mistook that for a subfolder of .swiftpm I can add it back in